### ISSUE TITLE
Remapping from mouse button1

### ISSUE THREAD

**Issue by mmxmb** [pqrs-org/Karabiner-Elements#2370]
Version: `12.10.0`

---

How can I remap **from** mouse `button1`? From key dropdown lists mouse buttons starting from `button2` onward for me. I can still map **to** mouse `button1`.

Editing this in`karabiner.json` directly doesn't work.

**Comment by kirintwn** [comment]
I'm able to achieve this by editing `karabiner.json` directly though. I swapped left & right button for my mouse.
```
"simple_modifications": [
  {
    "from": {
      "pointing_button": "button2"
    },
    "to": {
      "pointing_button": "button1"
    }
  },
  {
    "from": {
      "pointing_button": "button1"
    },
    "to": {
      "pointing_button": "button2"
    }
  }
]
```

Although it looks weird on UI, it works perfectly.
![image](https://user-images.githubusercontent.com/11881669/90360946-92704d80-e08f-11ea-8d55-a3fc143bfd67.png)

You might need to enable `modify events from this device` for your mouse beforehand (In the devices section of Karabiner config).
Reference: [karabiner-element official docs: How to change mouse buttons](https://karabiner-elements.pqrs.org/docs/help/how-to/mouse-button/)

**Comment by angelikatyborska** [comment]
I found this issue because I experienced the same problem. I'm right-handed and use a right-handed mouse, but I also use a trackball with my left hand whenever I need to let my right hand rest. I wanted to swap mouse buttons 1 and 2 just for the trackball, something that Karabiner-Elements is perfect for.

I discovered that this is possible via the UI after going into the "Expert" tab and enabling "unsafe configuration". Make sure to read the warning next to it - it's called "unsafe" for a reason. After adding the remapping, I was able to disable the "unsafe configuration" without losing the remapping.
